plugins/console/index.js

```diff
@@ -0,0 +1,127 @@
+var q = require('q');
+
+var testOut = {failedCount: 0, specResults: []};
```

Comment:
If I do that it will output "Console output" onto the console at the end of the test I think. That's what I am seeing. And pushing onto testOut.specResults.assertions causes an error.

However in its current form it seems to work. Maybe I am mistaken?